dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/communityManager/AggregatedDataRequestsTest.kt

```diff
@@ -0,0 +1,237 @@
+        )
+    }
+
+    @Test
+    fun `post bulk data request and check that the filter for reporting periods on aggregated level works properly`() {
+        val identifierMap = mapOf(DataRequestCompanyIdentifierType.lei to generateRandomLei())
+        val frameworks = listOf(BulkDataRequest.ListOfFrameworkNames.lksg)
+        val reportingPeriods = listOf("2020", "2021", "2022", "2023")
+        val response = requestControllerApi.postBulkDataRequest(
+            BulkDataRequest(identifierMap.values.toList(), frameworks, reportingPeriods),
+        )
+        checkThatAllIdentifiersWereAccepted(response, identifierMap.size)
+        val randomReportingPeriod = reportingPeriods.random()
+        val aggregatedDataRequests = requestControllerApi.getAggregatedDataRequests(
+            reportingPeriod = randomReportingPeriod,
+        )
+        iterateThroughFrameworksReportingPeriodsAndIdentifiersAndCheckAggregationWithCount(
+            aggregatedDataRequests, frameworks, listOf(randomReportingPeriod), identifierMap, 1,
+        )
+        reportingPeriods.filter { it != randomReportingPeriod }.forEach { filteredReportingPeriod ->
+            assertFalse(aggregatedDataRequests.any { it.reportingPeriod == filteredReportingPeriod })
+        }
+        val aggregatedDataRequestsWithoutFilter = requestControllerApi.getAggregatedDataRequests(reportingPeriod = null)
+        iterateThroughFrameworksReportingPeriodsAndIdentifiersAndCheckAggregationWithCount(
+            aggregatedDataRequestsWithoutFilter, frameworks, reportingPeriods, identifierMap, 1,
+        )
+        val aggregatedDataRequestsForEmptyString = requestControllerApi.getAggregatedDataRequests(reportingPeriod = "")
+        iterateThroughFrameworksReportingPeriodsAndIdentifiersAndCheckAggregationWithCount(
+            aggregatedDataRequestsForEmptyString, frameworks, reportingPeriods, identifierMap, 1,
+        )
+    }
+
+    @Test
+    fun `post bulk data request and check that the filter for request status on aggregated level works properly`() {
+        val randomLei = generateRandomLei()
+        val identifierMap = mapOf(DataRequestCompanyIdentifierType.lei to randomLei)
+        val frameworks = listOf(BulkDataRequest.ListOfFrameworkNames.lksg)
+        val reportingPeriods = listOf("2020", "2021")
+        val response = requestControllerApi.postBulkDataRequest(
+            BulkDataRequest(identifierMap.values.toList(), frameworks, reportingPeriods),
+        )
+        checkThatAllIdentifiersWereAccepted(response, identifierMap.size)
+        val aggregatedDataRequests = requestControllerApi.getAggregatedDataRequests()
+        assertNumberOfMatchesOnExclusivelyOpenRequestsEquals(aggregatedDataRequests, randomLei, 2)
+        val aggregatedRequestsNoFilter = requestControllerApi.getAggregatedDataRequests()
+        assertNumberOfMatchesOnExclusivelyOpenRequestsEquals(aggregatedRequestsNoFilter, randomLei, 2)
```

Comment:
Since the status is a nullable parameter of the `getAggregatedDataRequets` endpoint, I would filter once for a randomly picked status and once with an actual input of `null` for the status (see test above for reference)

Reply:
Gave it purpose again by giving a proper argument in line 199